calc.rb

```diff
@@ -0,0 +1,65 @@
+# Brandi Wilson - CALCULATOR (Lunch)
+
+# PROMPTS THE USER.
+puts "What do you want to do, Master?"
+operator_prompt = "\tAddition\tSubtraction\n\tMultiplication\tDivision\n\tExponent\tModulo"
+puts "#{operator_prompt}"   # so the user knows what they can enter
+user_operator = gets.chomp.upcase   # captures user input
+
+# ALL ACCEPTABLE INPUT (AS ARRAYS).
+addition = ["ADDITION", "ADD", "+"]
+subtraction = ["SUBTRACTION", "SUBTRACT", "SUB", "-"]
+multiplication = ["MULTIPLICATION", "MULTIPLY", "MULTI", "MULT", "*"]
+division = ["DIVISION", "DIVIDE", "DIV", "/"]
+modulo = ["MODULO", "MODULUS", "MOD", "REMAINDER", "%"]
+exponents = ["EXPONENTS", "EXPONENT", "EXP", "POWER", "THE POWER", "TO THE POWER", "^", "**"]
+all_operators = []   # creating a super long array
+all_operators.concat(addition).concat(subtraction).concat(multiplication).concat(division).concat(modulo).concat(exponents)
```

Comment:
If you're using `.concat`, it probably means to read through the Ruby array documentation, so :+1: for that.

__Protip__: Ruby lets you shortcut combining arrays by using `+`, so you could make this a little easier like this: 
```
all_operators = addition + subtraction + multiplication + division + modulo + exponents
```